README.md

```diff
@@ -1,16 +1,73 @@
 # SmallCity
 
-| Created by Abakir Hanna, Ethan Mooney, and Lindsey Weiskopf
-
+| Created by Abakir Hanna, Ethan Mooney, and Lindsey Weiskopf [SmallCity Live Demo](https://step2020-smallcity.appspot.com/)
 
 ### Objective
 
 We aim to create a web platform for users to find products from local small businesses. Our web app will increase exposure to local businesses of interest in their community. 
```

Comment:
Please remove the first line indents for these paragraphs.

Reply:
fixed